tools/tsp-client/package.json

```
@@ -1,14 +1,14 @@
 {
-  "name": "@azure-tools/tsp-client",
-  "version": "0.0.1",
+  "name": "@azure-tools/typespec-client-generator-cli",
```

```suggestion
  "name": "@azure-tools/typespec-client-generator",
```

Having `-cli` in the name seems redundant and inconsistent with other JS packages which provide a CLI.

---

When we were discussing the package name offline we reached the conclusion that adding `-cli` at the end it clearer. The name `@azure-tools/typespec-client-generator` implies that this package actually does the generation, but in reality it's only a tool to help folks generate client libraries.